opte/src/engine/icmpv6.rs

```diff
@@ -433,22 +431,26 @@ fn construct_neighbor_advert<'a>(
     }
 
     // Set the ROUTER flag, if required.
-    let mut flags = NdiscNeighborFlags::empty();
+    //
+    // Note from RFC 4861 Section 7.2.4 paragraph 2, we start with the OVERRIDE
+    // flag set. That says:
+    //
+    // > If the Target Address is either an anycast address or a unicast
+    // > address for which the node is providing proxy service, or the Target
+    // > Link-Layer Address option is not included, the Override flag SHOULD
+    // > be set to zero.  Otherwise, the Override flag SHOULD be set to one.
+    //
```

Comment:
Thanks for adding this comment with a callout to the RFC. Stuff like this helps greatly in future debugging when some pour soul is debugging an IPv6 support ticket.